### PR TITLE
remove old bip135 votes

### PR DESCRIPTION
**Pull request by gandrewstone**

### CODE REVIEW

File: src/consensus/params.h [hunk 1]
```diff
@@ -24,15 +24,16 @@ enum DeploymentPos
     // When a bit goes back to being unused, it should be renamed to
     // DEPLOYMENT_UNASSIGNED_BIT_x .
     // NOTE: Also add new deployments to VersionBitsDeploymentInfo in versionbits.cpp
-    DEPLOYMENT_CSV = 0, // bit 0 - deployment of BIP68, BIP112, and BIP113.
```
Comment (sickpig) [inline src/consensus/params.h:27]: Why bit 0 was initially assigned to BIP68, BIP112, and BIP113? Haven't we got thewm already activated by the time we introduced BIP135?

Comment (gandrewstone) [reply src/consensus/params.h:27]: yes, but we left it there because there's plenty of bits and it was previously specified as CSV

Comment (sickpig) [reply src/consensus/params.h:27]: ok.

**Comment by sickpig** [comment]
Shouldn't we also change `src/config/forks.csv`?

**Comment by gandrewstone** [comment]
yes, thanks, done

**Review by sickpig — APPROVED** [review]

**Review by ptschip — APPROVED** [review]
works with extended tests except bip68-sequence (which should be removed IMO anyway)

**Pull request merged**